perun-core/src/main/java/cz/metacentrum/perun/core/entry/AuditMessagesManagerEntry.java

```diff
@@ -47,15 +47,17 @@ public AuditMessagesManagerEntry() {
 
 	@Override
 	public List<AuditMessage> pollConsumerMessages(PerunSession perunSession, String consumerName) throws InternalErrorException, PrivilegeException {
-		if (!AuthzResolver.isAuthorized(perunSession, Role.PERUNADMIN)) {
+		if (!AuthzResolver.isAuthorized(perunSession, Role.PERUNADMIN)
+			&& !AuthzResolver.isAuthorized(perunSession, Role.PERUNOBSERVER)) {
```

Comment:
@zlamalp Do we really want to give an access to polling for perun observer? Because it is not just reading.

Reply:
@stavamichal you are right. This is a state changing method, while it doesn't look that way from the method name. @oernst9 please remove authorization for PERUNOBSERVER, thank you.